<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAddEvent"/>
    <addaction name="actionEditEvent"/>
    <addaction name="actionRemoveEvent"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionCharacters"/>
    <addaction name="actionLocations"/>
    <addaction name="actionCivilizations"/>
    <addaction name="actionTimelines"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewFile">
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="toolTip">
    <string>Create a new timeline file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>&amp;Import...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export...</string>
   </property>
  </action>
  <action name="actionViewCharacters">
   <property name="text">
    <string>&amp;Characters</string>
   </property>
  </action>
  <action name="actionViewLocations">
   <property name="text">
    <string>&amp;Locations</string>
   </property>
  </action>
  <action name="actionViewCivilizations">
   <property name="text">
    <string>Ci&amp;vilizations</string>
   </property>
  </action>
  <action name="actionViewEvents">
   <property name="text">
    <string>&amp;Timelines</string>
   </property>
  </action>
  <action name="actionViewRecent_Edits">
   <property name="text">
    <string>&amp;Recent Edits</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="actionProject_Metadata">
   <property name="text">
    <string>Project &amp;Metadata..</string>
   </property>
  </action>
  <action name="action_Minimize_Maximize">
   <property name="text">
    <string>&amp;Minimize / Maximize</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_Manual">
   <property name="text">
    <string>&amp;Manual...</string>
   </property>
  </action>
  <action name="actionAddEvent">
   <property name="text">
    <string>&amp;New Event...</string>
   </property>
  </action>
  <action name="actionRemoveEvent">
   <property name="text">
    <string>&amp;Remove Event...</string>
   </property>
  </action>
  <action name="actionAddCharacter">
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="toolTip">
    <string>Add a new character.</string>
   </property>
  </action>
  <action name="actionRemoveCharacter">
   <property name="text">
    <string>&amp;Delete...</string>
   </property>
   <property name="toolTip">
    <string>Delete a character</string>
   </property>
  </action>
  <action name="actionAddTimeline">
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="toolTip">
    <string>Create a new timeline.</string>
   </property>
  </action>
  <action name="actionRemoveTimeline">
   <property name="text">
    <string>&amp;Delete...</string>
   </property>
   <property name="toolTip">
    <string>Delete a timeline.</string>
   </property>
  </action>
  <action name="actionAddLocation">
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="toolTip">
    <string>Create a new location.</string>
   </property>
  </action>
  <action name="actionEditEvent">
   <property name="text">
    <string>&amp;Edit / View Event...</string>
   </property>
  </action>
  <action name="actionEditTimeline">
   <property name="text">
    <string>&amp;Edit / View...</string>
   </property>
  </action>
  <action name="actionEditCharacter">
   <property name="text">
    <string>&amp;Edit / View...</string>
   </property>
  </action>
  <action name="actionRemoveLocation">
   <property name="text">
    <string>&amp;Delete...</string>
   </property>
  </action>
  <action name="actionEditLocation">
   <property name="text">
    <string>&amp;Edit / View...</string>
   </property>
  </action>
  <action name="actionNewCivilization">
   <property name="text">
    <string>&amp;New...</string>
   </property>
  </action>
  <action name="actionRemoveCivilization">
   <property name="text">
    <string>&amp;Delete...</string>
   </property>
  </action>
  <action name="actionEditCivilization">
   <property name="text">
    <string>&amp;Edit / View...</string>
   </property>
  </action>
  <action name="actionViewEvent_Statistics">
   <property name="text">
    <string>&amp;Event Statistics</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close File</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCharacters">
   <property name="text">
    <string>&amp;Characters...</string>
   </property>
  </action>
  <action name="actionLocations">
   <property name="text">
    <string>&amp;Locations...</string>
   </property>
  </action>
  <action name="actionCivilizations">
   <property name="text">
    <string>Ci&amp;vilizations...</string>
   </property>
  </action>
  <action name="actionTimelines">
   <property name="text">
    <string>&amp;Timelines...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
